src/main/resources/com/suse/matcher/rules/drools/InputAugmenting.drl

```diff
@@ -38,3 +38,101 @@ rule "generatePenaltyGroups"
         insert(new PenaltyGroup(penaltyGroupId, $guestId));
 end
 
+// aggregate 2 equal (same properties, same product set) OEM subscriptions into a single one
+rule "aggregateEqualStackableSubscriptions"
+    agenda-group "InputAugmenting"
+    when
+        $s1: Subscription(ignored == false, stackable == true, id < 0, aggregationSubscriptionId == null || aggregationSubscriptionId == id)
+
+        // make sure that the $s1 is the one that has lowest id
+        not Subscription(
+            ignored == false,
+            stackable == true,
+            id < 0, // OEM subscription
```

Comment:
Totally optional: one possibility to make this more readable is to have an oem boolean property that gets populated via `InputAugmenting.drl` :wink:

Reply:
We are on a thin ice here. Who knows when somebody reports that they need such a rule for regular subscriptions in small quantities. I'd prefer to keep it this way.